### ISSUE TITLE
Consider publishing more mods as Github Releases?

### ISSUE THREAD

**Issue by Emily1799** [Shockah/Stardew-Valley-Mods#63]
Would it be possible to upload .zip files for your mods here, so that a Nexus account isn't required to download? Or perhaps to https://forums.stardewvalley.net/resources/

**Comment by Shockah** [comment]
Hey. I wasn't planning on releasing the mods anywhere outside of NexusMods. Doing Github releases on a monorepo is a total pain in the ass and wrong. The two mods I did release that way are there, because they're dependencies for Stardew Access, an accessibility mod, and Nexus isn't really an accessible website. The forums are also yet another place I'd have to update manually, and honestly, I've been modding Stardew for several years now, and have never heard of anyone releasing mods on the forums (other than unofficial updates), or looking for mods there.
Not that anything I say here has any merit now, as funnily enough, today I announced on the Stardew Valley Discord server that I'm stepping back from modding this game, effective immediately, due to burnout. People are free to contact me if they want to continue working on some/all of my mods, mostly to discuss granting NexusMods page editing permissions, switching DP gain to the new authors, maybe granting collaborator access to this repo (although this being a monorepo makes it problematic if a new author wants to only work on one of the mods).
